translations/german.json

```diff
@@ -498,132 +498,132 @@
-		"Register": "",
-		"The user '${targetUsername}' was not found.": "",
-		"You are locked and cannot challenge unlocked users.": "",
-		"You are banned from battling and cannot challenge users.": "",
-		"You must choose a username before you challenge someone.": "",
-		"This server requires you to be rank ${groupName} or higher to challenge users.": "",
-		"This command does not support specifying multiple users": "",
-		"User \"${targetUsername}\" not found.": "",
-		"Provide a valid format.": "",
-		"Please provide a valid format.": "",
-		"The format '${originalFormat.name}' was not found.": "",
-		"Your team is valid for ${format.name}.": "",
-		"Your team was rejected for the following reasons:": "",
-		"Battles are now hidden (except to staff) in your trainer card.": "",
-		"Battles are now visible in your trainer card.": "",
-		"'${command}' is a help command.": "",
-		"The command '/${target}' does not exist.": "",
-		"Could not find help for '/${target}'. Try /help for general help.": "",
-		"Could not find help for '/${target}' - displaying help for '/${closestHelp}' instead": ""
+		"Server version: <b>${version}</b>": "Server-Version: <b>${version}</b>",
+		"/mee - must not start with a letter or number": "/mee - darf nicht mit einem Buchstaben oder einer Zahl anfangen",
+		"What?! How are you not more excited to battle?! Try /battle! to show me you're ready.": "Wie bitte?! Wie kannst du dich nicht über einen Kampf freuen?! Sobald du bereit bist, benutze den Befehl /battle!",
+		"Access denied for custom avatar - make sure you're on the right account?": "Zugriff auf einen Custom Avatar verweigert - bitte stelle sicher, dass du den richtigen Account verwendest.",
+		"Invalid avatar.": "Ungültiger Avatar.",
+		"Avatar changed to:": "Avatar geändert zu:",
+		"Artist: ": "Ersteller: ",
+		"No one has PMed you yet.": "Niemand hat dir bisher eine private Nachricht geschrieben.",
+		"You forgot the comma.": "Du hast das Komma vergessen.",
+		"User ${targetUsername} not found. Did you misspell their name?": "Der Nutzer ${targetUsername} wurde nicht gefunden. Hast du den Namen falsch geschrieben?",
+		"User ${targetUsername} is offline.": "Der Nutzer ${targetUsername} ist offline.",
+		"The user \"${targetUsername}\" was not found.": "Der Nutzer \"${targetUsername}\" wurde nicht gefunden.",
+		"The room \"${target}\" was not found.": "Der Raum \"${target}\" wurde nicht gefunden.",
+		"You do not have permission to invite people into this room.": "Du hast keine Berechtigung, Nutzer in diesen Raum einzuladen.",
+		"This user is already in \"${targetRoom.title}\".": "Dieser Nutzer ist bereits in \"${targetRoom.title}\".",
+		"Setting status messages in /busy is no longer supported. Set a status using /status.": "Es ist nicht mehr möglich, eine Statusmeldung einzustellen, solange /busy aktiviert ist.",
```

Comment:
Shouldn't this message mention the `/status` command?

Reply:
Oh yikes, completely overlooked the second sentence, added it; thank you for pointing this out Annika